bigquery/data-transfer/api/QuickStart/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Copyright (c) 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START bigquery_data_transfer_quickstart]
+
+using System;
+using Google.Api.Gax;
+using Google.Cloud.BigQuery.DataTransfer.V1;
+
+namespace QuickStart
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Instantiates a client
+            DataTransferServiceClient client = DataTransferServiceClient.Create();
+
+            // Your Google Cloud Platform project ID
+            string projectId = "YOUR-PROJECT-ID";
+
+            ProjectName project = new ProjectName(projectId);
+            PagedEnumerable<ListDataSourcesResponse, DataSource> sources = client.ListDataSources(ParentNameOneof.From(project));
+            Console.WriteLine("Supported Data Sources:");
+            foreach (DataSource source in sources)
+            {
+                Console.WriteLine($"{source.DataSourceId}: {source.DisplayName} ({source.Description})");
+            }
+            Console.WriteLine("\nPress any key...");
+            Console.ReadKey();
```

Comment:
I see that https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/47aa6ac0ad2b9db68c484828441cfc583ee341f7/bigquery/api/BigquerySample/Program.cs contains a call to ReadKey(), so you followed the pattern and added it here.

But, we have stopped calling ReadKey() at the end of quick starts.  The BigquerySample was the last left over.  Please remove it here.

Reply:
Done.